pkg/network/usm/sharedlibraries/ebpf.go

```diff
@@ -38,11 +43,31 @@ const (
 
 var traceTypes = []string{"enter", "exit"}
 
+// We cannot use sync.Once as we need to reset the singleton
```

Comment:
where do we need to reset?
If that's in tests, then we can reset the `Once` object in the tests

Reply:
I think it's only in tests, as I don't believe we can disable and reenable modules during runtime in system-probe. However, as the `UprobeAttacher` is also using this program under the hood, we'd need to add the cleanup to all those tests, so I thought this was a better alternative.